### PR TITLE
Problem comment reactions

### LINKED ISSUE #452: Problem comment reactions

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #452

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
https://github.com/taskany-inc/hire/assets/81121562/3eddc80c-4a6c-4d8b-a2ee-91bf58086f4b

**Comment by Katrin-kudryash** [comment]
https://github.com/taskany-inc/hire/assets/81121562/96056e72-f366-489a-8870-459a17e3969a

**Comment by Katrin-kudryash** [comment]
https://github.com/taskany-inc/hire/assets/81121562/4b49168e-51c3-461b-af22-f5c417335e63

File: src/modules/problemMethods.ts [hunk 1]
```diff
@@ -134,15 +135,56 @@ const getById = async (id: number) => {
...
+                    user: true,
+                    attaches: true,
+                    reactions: { include: { user: { select: { name: true, email: true } } } },
+                },
+            },
         },
     });
 
+    const limit = 10;
+    const comments = problem?.comments.map((comment) => {
```
Comment (tsumo) [inline src/modules/problemMethods.ts:149]: comment formatting could be moved to utils

File: src/utils/pageContext.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import React from 'react';
+
+export interface PageContext {
+    theme: 'light' | 'dark';
+    themeId: number;
+    ssrTime: number;
+    user?: Session['user'];
+}
+
+export const pageContext = React.createContext<PageContext>({ theme: 'dark', themeId: 0, ssrTime: 0 });
```
Comment (tsumo) [inline src/utils/pageContext.ts:11]: This context is not rendered anywhere, it will not work

Better to do this in separate issue

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**